gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/infra/adapter/ApiAdapter.java

```diff
@@ -53,6 +53,12 @@ public interface ApiAdapter {
     Stream<io.gravitee.repository.management.model.Api> toRepositoryStream(Stream<Api> source);
 
     @Mapping(target = "apiVersion", source = "version")
+    @Mapping(target = "tags", source = "source.apiDefinitionV4.tags")
```

Comment:
If there is only one parameter, no need to specify `source.` in the source value